setup-slave.sh

```diff
@@ -19,13 +19,12 @@ hostname $PRIVATE_DNS
 echo $PRIVATE_DNS > /etc/hostname
 HOSTNAME=$PRIVATE_DNS  # Fix the bash built-in hostname variable too
 
-echo "Setting up slave on `hostname`..."
-
-# Work around for R3 instances without pre-formatted ext3 disks
+# Work around for R3 or I2 instances without pre-formatted ext3 disks
 instance_type=$(curl http://169.254.169.254/latest/meta-data/instance-type 2> /dev/null)
-if [[ $instance_type == r3* ]]; then
-  # If already exists, umount
-  umount /mnt
```

Comment:
@filipenf Eles recomendam não fazer isso. Em http://spark.apache.org/docs/latest/hardware-provisioning.html:
We recommend having 4-8 disks per node, configured **without** RAID (just as separate mount points). In Linux, mount the disks with the noatime option to reduce unnecessary writes. In Spark, configure the spark.local.dir variable to be a comma-separated list of the local disks. If you are running HDFS, it’s fine to use the same disks as HDFS.